Packs/CrowdStrikeMalquery/Integrations/CrowdStrikeMalquery/CrowdStrikeMalquery.yml

```diff
@@ -0,0 +1,358 @@
+    outputs:
+    - contextPath: Malquery.Request_ID
+      description: The request ID.
+      type: String
+    - contextPath: Malquery.Status
+      description: The request status.
+      type: String
+    description: 'Schedule samples for download. Using samples-multidownload is a
+      three-step process: 1. Schedule the download with samples-multidownload, which
+      returns a request ID. 2. Provide that request ID to the cs-malquery-get-request,
+      in order to check the status of the operation. 3. When the request status is
+      “done”, use cs-malquery-sample-fetch to download the results as a password-protected
+      archive. The password to extract results from the archive: infected'' '
+  - name: cs-malquery-file-download
+    arguments:
+    - name: file_id
+      required: true
+      description: The file SHA256.
+    description: Download a file indexed by MalQuery. Specify the file using its SHA256.
+      Only one file is supported at this time.
+    outputs:
+    - contextPath: File.SHA256
+      description: SHA256 hash of the file.
+      type: String
+    - contextPath: File.SHA1
+      description: SHA1 hash of the file.
+      type: String
+    - contextPath: File.SHA512
+      description: SHA512 hash of the file.
+      type: String
+    - contextPath: File.Name
+      description: Name of the file.
+      type: String
+    - contextPath: File.EntryID
+      description: Entry ID.
+      type: String
+    - contextPath: File.Info
+      description: File info.
+      type: String
+    - contextPath: File.Type
+      description: Type of the file.
+      type: String
+    - contextPath: File.MD5
+      description: MD5 hash of the file.
+      type: String
+  - name: cs-malquery-sample-fetch
+    arguments:
+    - name: request_id
+      required: true
+      description: Identifier of a MalQuery request.
+    description: Fetch a zip archive with password infected containing the samples.
+      Call this once the cs-malquery-samples-multidownload request has finished processing
+    outputs:
+      - contextPath: File.SHA256
+        description: SHA256 hash of the file.
+        type: String
+      - contextPath: File.SHA1
+        description: SHA1 hash of the file.
+        type: String
+      - contextPath: File.SHA512
+        description: SHA512 hash of the file.
+        type: String
+      - contextPath: File.Name
+        description: Name of the file.
+        type: String
+      - contextPath: File.EntryID
+        description: Entry ID.
+        type: String
+      - contextPath: File.Info
+        description: File info.
+        type: String
+      - contextPath: File.Type
+        description: Type of the file.
+        type: String
+      - contextPath: File.MD5
+        description: MD5 hash of the file.
+        type: String
+  - name: file
+    arguments:
+    - name: file
+      required: true
+      description: The file SHA256.
+      isArray: false
```

Comment:
should support multiple values

Reply:
@yaakovi - talked about it with @Arsenikr. He told me that although the API allows it, the command should only support 1 file at a time.